datacube_stats/main.py

```diff
@@ -137,8 +140,8 @@ def identity(f):
 @click.option('--version', is_flag=True, callback=_print_version,
               expose_value=False, is_eager=True)
 @ui.pass_index(app_name='datacube-stats')
-def main(index, stats_config_file, qsub, runner, save_tasks, load_tasks,
-         tile_index, tile_index_file, output_location, year, task_slice, batch):
+def main(index, stats_config_file, save_tasks, load_tasks,
+         tile_index, tile_index_file, output_location, year, task_slice, batch, runner=identity, qsub=None):
```

Comment:
I did not realize it is *this* simple to fix it. :+1:

Reply:
Whoops, I haven't tested that properly - I didn't clean my git stash properly before opening a new branch.